tests/test_qubes_rpc.py

```diff
@@ -1,14 +1,55 @@
 import os
+import subprocess
 import unittest
 
+RETURNCODE_SUCCESS = 0
+RETURNCODE_DENIED = 126
+
 
-# TODO: add actual integration testing of policies
 class SD_Qubes_Rpc_Tests(unittest.TestCase):
-    def test_policies(self):
+    def _qrexec(self, source_vm, dest_vm, policy_name):
+        cmd = ["qvm-run", "--pass-io", source_vm, f"qrexec-client-vm {dest_vm} {policy_name}"]
+        p = subprocess.run(cmd, input="test", capture_output=True, text=True, check=False)
+        return p.returncode
+
+    def test_policies_exist(self):
         """verify the policies are installed"""
         assert os.path.exists("/etc/qubes/policy.d/31-securedrop-workstation.policy")
         assert os.path.exists("/etc/qubes/policy.d/32-securedrop-workstation.policy")
 
+    # securedrop.Log from @tag:sd-workstation to sd-log should be allowed
+    def test_sdlog_from_sdw_to_sdlog_allowed(self):
+        self.assertEqual(self._qrexec("sd-app", "sd-log", "securedrop.Log"), RETURNCODE_SUCCESS)
```

Comment:
Would it make sense to use qubesadmin here to get the sets of VMs tagged with `sd-workstation` (and not, for tests that check for blocking policies) and use those in the tests instead of hard-coding VMs?